tests/resources/xml/testValidServiceUrl.xml

```diff
@@ -2,8 +2,11 @@
   <cas:authenticationSuccess>
     <cas:user>testuser@example.com</cas:user>
     <cas:attributes>
+      <cas:authenticationDate>2024-11-14T14:44:21Z</cas:authenticationDate>
```

Comment:
They are defined in the xsd-schema for CAS 3.0 and they are mandatory.
https://github.com/simplesamlphp/xml-cas/blob/master/resources/schemas/cas-server-protocol-3.0.xsd#L92-L111

Reply:
> I think they are getting added by xml-cas

This is correct.

The reason why we see these new attributes is because i upgraded the xml-cas to the version 1.3.x. I checked the code. What do you want me to do? Is this a breaking change or it just introduces more attributes that you can use but the login flow works as before?

Reply:
I think leave them in, and we (Cirrus) will need to get more real world data on the impact. Currently in places where we are a cas client, the CAS servers do not release those attributes to us.  In places we are a casserver we don't currently release those attributes.    Our real world experience has tended to be the spec and what other clients or server implemented have only partial alignment.